main.nf

```diff
@@ -869,10 +871,13 @@ if (!params.skip_alignment) {
     ch_samplesheet_reformat
         .splitCsv(header:true, sep:',')
         .map { get_sample_info(it, params.genomes) }
-        .map { it -> if (it[1].toString().endsWith('.bam')) [ it[0], it[1] ] }
-        .set { ch_sortbam_quant }
+        .map { it -> if (it[1].toString().endsWith('.bam')) [ it[0] , it[1] ] }
+        .into { ch_sortbam_quant
+                ch_sample_map_deseq2
+                ch_sample_map_dexseq }
```

Comment:
```suggestion
        .into { 
            ch_sortbam_rename
            ch_sortbam_quant
        }
 
 Hmmm...yes. Good point 🤔 Maybe the most consistent thing to do is to add an additional process to rename the BAM files [here](https://github.com/nf-core/nanoseq/blob/85f2ef340183096f34b9a30e692be978b4769cb6/main.nf#L873)? and then pass those files to the downstream processes?

Something like this:

```
process BAM_RENAME {
    tag "$sample"

    input:
    tuple val(sample), path(bam) from ch_sortbam_rename

    output:
    tuple val(sample), path("*.bam") into ch_sortbam_quant

    script:
    """
    [ ! -f ${sample}.bam ] && ln -s $bam ${sample}.bam
    """
}
```
```

Reply:
Hmmm...yes. Good point 🤔 Maybe the most consistent thing to do is to add an additional process to rename the BAM file like I have done above by adding a small process, and then pass those files to the downstream processes?

Reply:
Hi @drpatelh, thank you for the modification! We have added the process you wrote to the pull request and modified the Rscripts for DESeq2 and DEXSeq.